requirements.txt

```diff
@@ -1,3 +1,4 @@
 django==1.9.5,
 mysql-python==1.2.5,
 django-taggit==0.18.1,
+pytz==2016.4
```

Comment:
Is 2016.4 the version number? Please use a version number here and use the same version number in the travis.yml